### PR TITLE
Feat: P0 Screens

### PR DESCRIPTION
**Pull request by anandsr21harshit**
- Added P0 screens
- Fully responsive screens
- Used mobile first approach
- Used CSS Modules
- Use "/" , "/home", "/profile" to check pages

### CODE REVIEW

**Comment by anandsr21harshit** [comment]
Depoyed again and resolved it.
Check demo here: https://unique-manatee-a7bc7a.netlify.app/

File: src/frontend/pages/home/Home.module.css [hunk 1]
```diff
@@ -0,0 +1,12 @@
+.container{
+    margin: 5rem 0;
+    flex-direction: column;
+    align-items: center;
+}
+
+@media screen and (min-width: 500px) {
+    .container{
+        max-width: 500px;
```
Comment (chanchal-panpaliya) [inline src/frontend/pages/home/Home.module.css]: use rem or vw

File: src/frontend/components/NewComment/NewComment.module.css [hunk 2]
```diff
@@ -0,0 +1,51 @@
+.container{
+    gap:1rem;
+    margin-top: 1rem;
+    min-height: 2rem;
+    width: 100%;
+    /* border: 1px solid; */
```
Comment (chanchal-panpaliya) [inline src/frontend/components/NewComment/NewComment.module.css]: remove comment

**Review by chanchal-panpaliya — APPROVED** [review]
code looks clean with proper naming and good UI

File: src/frontend/components/Comment/Comment.jsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import CommentCSS from "./Comment.module.css"
+// import man from "../../assets/man.png"
```
Comment (manish591) [inline src/frontend/components/Comment/Comment.jsx]: Remove comment

File: src/frontend/components/Comment/Comment.module.css [hunk 4]
```diff
@@ -0,0 +1,10 @@
+.container{
+    margin-top: 1rem;
+    /* border: 1px solid; */
```
Comment (manish591) [inline src/frontend/components/Comment/Comment.module.css]: Remove comment

File: src/frontend/components/NewComment/NewComment.module.css [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+    width: calc(100% - 3rem);
+    border-radius: 0.5rem;
+    align-items: center;
+    justify-content: space-between;
+    padding: 0 0.5rem;
+    position: relative;
+}
+
+.input_area{
+    /* border: 1px solid; */
```
Comment (manish591) [inline src/frontend/components/NewComment/NewComment.module.css:25]: Remove comment

File: src/frontend/components/ProfileCard/ProfileCard.jsx [hunk 6]
```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { User } from '../User/User'
+import ProfileCardCSS from "./ProfileCard.module.css"
+import linkedin from "../../assets/linkedin.png"
+import github from "../../assets/github.png"
+import twitter from "../../assets/twitter.png"
+// import { PostCard } from '../PostCard/PostCard'
```
Comment (manish591) [inline src/frontend/components/ProfileCard/ProfileCard.jsx]: remove comment

File: src/frontend/components/ProfileCard/ProfileCard.jsx [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+        </div>
+        <article className={ProfileCardCSS.description}>
+            Aspiring Front-End Developer
+        </article>
+        <ul className={`d-flex ${ProfileCardCSS.stats}`}>
+            <li>3 Posts</li>
+            <li>5 Followers</li>
+            <li>2 Following</li>
+        </ul>
+        <a href="https://www.linkedin.com/in/hanand2021" target={"_blank"} className={ProfileCardCSS.social_media}>
```
Comment (manish591) [inline src/frontend/components/ProfileCard/ProfileCard.jsx]: No need to add curly braces around `_blank`

**Review by manish591 — APPROVED** [review]
The code is looking good. 
In Some places, indentation is not consistent. Use prettier.

File: src/frontend/components/ProfileCard/ProfileCard.jsx (hunk 7 above)
Comment (anandsr21harshit) [reply src/frontend/components/ProfileCard/ProfileCard.jsx]: Okay. Will make the changes as suggested

**Pull request merged**